hbogolib/handlereu.py

```diff
@@ -869,8 +876,91 @@ def categories(self):
         else:
             self.log("No Home Category found")
 
+        self.addCat('Last 10 Watched', '{0}?mode=1&action=listing&category={1}'.format(self.base_url, 'Last 10 Watched'),
```

Comment:
Label hardcoded should be id from the language file, see how My List is implemented you can implement it in the same way... same for the rest, the list action can be simply used for this... no need for additional actions and methods that are basically clones of the list method, this, in the long run, would be bad for maintainability.